src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -1435,6 +1435,12 @@ public static bool IsInstanceContext(this SyntaxTree syntaxTree, int position, S
                 return IsInBlock(accessor.Body, position);
             }
 
+            var propertyDeclaration = containingMember as PropertyDeclarationSyntax;
```

Comment:
Correct. They could be methods, operators, conversions, indexers in addition to being properties. 
[see design notes on codeplex](https://roslyn.codeplex.com/discussions/540574)